tests/bootstrap.php

```diff
@@ -6,8 +6,13 @@
 
 require_once __DIR__.'/../../../lib/base.php';
 
+\OC::$composerAutoloader->addPsr4('Test\\', OC::$SERVERROOT . '/tests/lib/', true);
```

Comment:
you can fully drop this file and use core's bootstrap file

Reply:
I have errors:
```
OCP\AutoloadNotAllowedException: Autoload path not allowed: /Users/mrow4a/Projects/www/octest/apps/richdocuments/lib/appinfo/application.php
/Users/mrow4a/Projects/www/octest/lib/autoloader.php:133
/Users/mrow4a/Projects/www/octest/lib/autoloader.php:157
/Users/mrow4a/Projects/www/octest/apps/richdocuments/tests/unit/appinfo/applicationtest.php:31
```